src/components/Tab/Tab.tsx

```diff
@@ -33,7 +35,8 @@ function Tab(props: TabProps) {
     isSubTab = false,
     navigateTo,
     info,
-    hasIncompleteFields
+    hasValidIndicator = false,
+    unfilledFields
```

Comment:
Let's make the name a bit more generic, e.g. `invalidFieldsLength` or something like that